### ISSUE TITLE
Include google form as an option to add a conference/meeting to the calendar

### ISSUE THREAD

**Issue by smurgia** [LSSTDESC/CosmologyMeetings#21]
**Is your feature request related to a problem? Please describe.**
The DESC speakers bureau has discussed having a google form as a way to add a conference/meeting to the cosmology calendar. 

**Describe the solution you'd like**
Add the link to the google form to the cosmology meetings space. Here's the link to the form: https://forms.gle/UJFR257HPQSFqayb9
This can be added to the top-left corner, in addition to the GitHub option. Also, it should be mentioned in the text, i.e. "If you know of a meeting that should be on this calendar, please [post an issue here]" should be replaced with "If you know of a meeting that should be on this calendar, please [post an issue here] or fill out this [form](https://forms.gle/UJFR257HPQSFqayb9)." 

**Describe alternatives you've considered**
N/A

**Additional context**
The Google form allows to fill the calendar semi-automatically (a person is still involved in the process the evaluate the request.) Members of the SB can have access to it.

**Comment by rmandelb** [comment]
I should preface my comment by saying I don't really have a horse in this race, and I realize the SB may have other considerations besides the ones I'm thinking of here.  So feel free to ignore me, but these are just my reactions based on my experience trying to administer something similar in the past: 

SB members have access to this repo, as well, and can subscribe to be notified via email of new issues, so it seems similar to a google form from an access perspective.  

And GitHub issues seem even more convenient from the standpoint of iteration -- if you keep using the issues, SB members who want to add the event to the calendar can ask questions directly in the thread rather than having to initiate an email conversation with whoever made the request.  The GitHub issue is also a place to simply acknowledge the request without having to initiate a new communication.

I am a little concerned that having two ways to do the same thing will end up being more work (and more confusing) than having just one, and the google form option seems like more work (due to its one-way nature) than GitHub issues.

**Comment by smurgia** [comment]
Thank you Rachel for the comments. I didn't think about the opportunity for comments, and I agree that having two ways of submitting requests is not ideal. If possible, I would modify the submission form on GitHub to include more guidance for the submission, along the lines of what is in the google form. In addition to conference/meeting name and dates, the submitter should could also specify other useful information, e.g. meeting type, url, abstract submission deadline, etc.  Let me know what you think. I'm happy to work on it.

**Comment by rmandelb** [comment]
Yes, it's easy to update the issue template so that it specifies much more about what information people should include (https://github.com/LSSTDESC/CosmologyMeetings/blob/master/.github/ISSUE_TEMPLATE/new-meeting.md).  If you are familiar with how to do this, you should go ahead and make a PR for review by members of the Speakers Bureau.  I'm also happy to review it.  If you would like help with it, I'm happy to help - just let me know.

**Comment by smurgia** [comment]
Thanks for the info. I've updated the form. I only included a field for abstract submission deadline, and separate start and end date. Please have a look if you get a chance.
